webapp/tests/model/test_form_data.py

```diff
@@ -208,11 +208,11 @@ def test_if_all_data_is_provided_then_fill_familienstand_correctly(self):
 
     def test_if_show_person_b_false_then_raise_no_error_if_person_b_fields_missing(self):
         with patch('app.model.form_data.FamilienstandModel.show_person_b', MagicMock(return_value=False)):
-            MandatoryFormData.parse_obj({**self.valid_data_person_a, **self.married_familienstand, **self.valid_steuernummer})
+            MandatoryFormData.parse_obj({**self.valid_data_person_a, **self.married_familienstand, **self.valid_steuernummer, **{'is_user_account_holder': 'yes'}})
```

Comment:
Why isn't this part of the self.married_familienstand object? Because it semantically doesn't fit?